### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ScreenReader:Screen reader is announcing the error message as in table format, which is not visually representing as table.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17196]
## Actual:
Screen reader is announcing the error message as in table format, which is not visually representing as table.

**Similar issue is observed with Jaws**

## Expected:
The format of the error message should be either given as plain text or the error message should be structured with proper UI of table, with column headers and proper alignments.

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:text;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Screen Reader: NVDA(2021.3), Jaws(2021.2103.174)

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Press enter “New Query” write any valid query and click on "Execute query button"
7. Message Window will open
8. Run NVDA and tab to the link "line" in error message and check if the NVDA is announcing the error message as table.

## User Impact:
Screen reader users will not get the proper information about the error message.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
## Attachments
![17196](https://user-images.githubusercontent.com/86601127/149121661-9e9e934f-dbd7-44ca-8436-cdcf0e927e39.gif)
![17196 2](https://user-images.githubusercontent.com/86601127/149122378-323d5e37-52bd-4a1a-8303-fcda0b3012f8.gif)

**Comment by smartguest** [comment]
This is not an error message @Roopesh-T This is just a normal message. 

Clicking the link will lead directly to the first line of the start of the query.

I do agree that its best to probably change the text to something a bit more descriptive and general instead of just listing the table format, rather than representing the table visually again as it's already available, just in another tab.

**Comment by smartguest** [comment]
Closing as this is by design that we use a table to organize the messages, so this readout is expected.

**Comment by Roopesh-T** [comment]
@smartguest The issue is not repro in latest env 
![216575569-acdc8f4a-0d0c-4911-8672-31a37b6a4167](https://user-images.githubusercontent.com/86601127/216590981-3d3405cf-49d4-40e4-8c3f-82cce47aaa4c.png)

![17196_Closed](https://user-images.githubusercontent.com/86601127/216592203-58534277-e64d-4775-8be4-f757480a1708.gif)
